package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@web3-ui/root",
   "license": "MIT",
   "version": "0.0.0",
-  "private": "true",
+  "private": "false",
```

Comment:
We want this to stay private, right? This should never be published because it's just a  way to coordinate the other packages.